### PR TITLE
[bugfix] mapbox pan on filter

### PR DESCRIPTION
**Pull request by xtinec**
- Modify the backend to send back the bounds of the points to cluster
- Clean up of mapbox component to derive lat, lon and zoom from the mercator that fitsbounds based on the bounds passed from the backend. This and the computation of clusters are now only done on initial load b/c we are not making queries to the backend on pan/zoom
- A minor clean up of unused props and also making the visual properties panel open by default
- Removal of default lat, lon and zoom as they are no longer needed to compute the viewport on render.

@mistercrunch @betodealmeida @youngyjd

### CODE REVIEW

File: superset/assets/src/utils/common.js [hunk 1]
```diff
@@ -9,9 +9,6 @@ export const LUMINANCE_RED_WEIGHT = 0.2126;
 export const LUMINANCE_GREEN_WEIGHT = 0.7152;
 export const LUMINANCE_BLUE_WEIGHT = 0.0722;
 export const MILES_PER_KM = 1.60934;
-export const DEFAULT_LONGITUDE = -122.405293;
```
Comment (kristw) [inline superset/assets/src/utils/common.js]: ❤️

File: superset/assets/src/visualizations/MapBox/MapBox.jsx [hunk 2]
```diff
@@ -6,12 +6,6 @@ import Immutable from 'immutable';
 import supercluster from 'supercluster';
 import ViewportMercator from 'viewport-mercator-project';
 import ScatterPlotGlowOverlay from './ScatterPlotGlowOverlay';
-
-import {
```
Comment (kristw) [inline superset/assets/src/visualizations/MapBox/MapBox.jsx]: Thank you for removing!

File: superset/viz.py [hunk 3]
```diff
@@ -2015,6 +2015,9 @@ def get_data(self, df):
             ],
         }
 
+        southWest = [df[fd.get('all_columns_x')].min(), df[fd.get('all_columns_y')].min()]
```
Comment (mistercrunch) [inline superset/viz.py]: NIT: snake over camel case in python

**Review by kristw — APPROVED** [review]
👍

File: superset/viz.py (hunk 3 above)
Comment (mistercrunch) [reply superset/viz.py]: NIT (optional): you could save some hash lookups by going `x_series = df[fd.get('all_columns_x')]` first

**Review by mistercrunch — CHANGES_REQUESTED** [review]
Some minor nits, otherwise LGTM.

As discussed we should eventually deprecate this viz and resolve the github fork ref in `package.json`. The `deck_` are superior, and have things like a `ViewportControl`, support the "time player", ...

**Review by mistercrunch — APPROVED** [review]

**Pull request merged**